### DISCUSSION TITLE
Feedback on dashboard designs

### DISCUSSION THREAD

**Discussion by swedishfrenchpress** [cashubtc/eNuts#144] [Ideas]
Hey team, I recently conducted an A/B design test on Twitter to gather user feedback on two of our design variations. Below is the organized feedback from the respondents to help guide our next steps in the development process. It's important that we just keep in mind that this is feedback and we want to avoid [designing by committee](https://en.wikipedia.org/wiki/Design_by_committee). 

![1](https://github.com/cashubtc/eNuts/assets/78821053/e82fad30-86da-4687-ad30-73397779b994)

## **1. Tally of Preferences:**
- **Design A**: 6 votes
- **Design B**: 5 votes

### **Design A:**
1. **Functionality and Information**: 
   - "I want information and quick access, not unnecessary blank space."
   - "I want to see the most recent transactions immediately without extra clicking."
   - "A seems more useful, as the user does not have to hunt for history in another view. I believe this is expected for any modern fintech app."

2. **Preference for A without Specific Reason**: 
   - Three users simply mentioned they preferred "A" without providing a specific reason.

### **Design B:**
1. **Simplicity and Cleanliness**: 
   - "B is nice and simple. Less intimidating look."
   - "B. Looks cleaner and less crowded."
   - "B is cleaner."

2. **Interest in Features**: 
   - "Does the 'Mints' show all the different balances kinda like different accounts in a traditional banking app?"

3. **General Preference for B**: 
   - One user mentioned they preferred "B" without giving a specific reason.

## **3. Questions & Uncertainties:**
1. **User Interaction Context**:
   - "When does the user see this screen? On log in?"

2. **Feature Clarifications**:
   - "What does “mints” do?"
   - "What does the QR logo do?"

So, we have some potential areas of improvement. For next steps I would suggest refining the designs based on this feedback and possibly conducting another round of testing or feedback to validate any changes made.

**Comment by KKA11010** [comment]
Lovely 💪 very helpful!

**Reply by swedishfrenchpress** [reply]
Keep in mind that these results don't include the poll findings here: https://twitter.com/realmuster/status/1694595402488496195

Poll closes in 10 hours. I'll revisit and update once the poll closes.

**Comment by KKA11010** [comment]
This discussion can be closed now
